docs/www/support.md

```diff
@@ -192,3 +192,13 @@ any of the remaining nodes will automatically assume the cluster leadership.
 
 Make sure that the installed certificates haven't expired. If they have,
 generate new ones and restart the node.
+
+### Replication factor issues
+
+If you are setting up a [Setup a 3-node cluster](https://vectorized.io/docs/quick-start-docker#Set-up-a-3-node-cluster) and create a topic with 2 replicas like so:
```

Comment:
```suggestion
If you are setting up a [Setup a 3-node cluster](https://vectorized.io/docs/quick-start-docker#Set-up-a-3-node-cluster) and create a topic with two replicas like the following, then the following command would fail because `redpanda` is using `Raft for concensus`:
```

Reply:
```suggestion
If you are setting up a [Setup a 3-node cluster](https://vectorized.io/docs/quick-start-docker#Set-up-a-3-node-cluster) and create a topic with two replicas like the following, then the following command would fail because `rpk` is using `Raft Consensus`:
```

Reply:
Should we also add this link for Raft?

Reply:
Good idea @Deflaimun